### PR TITLE
fix: schedule & details resolve bugs

### PR DESCRIPTION
**Pull request by ruzniaievdm**
[2U-244](https://youtrack.raccoongang.com/issue/2U-244)

### CODE REVIEW

File: src/schedule-and-details/index.jsx [hunk 1]
```diff
@@ -309,6 +309,7 @@ const ScheduleAndDetails = ({ intl, courseId }) => {
             <StatefulButton
               onClick={handleUpdateValues}
               state={isQueryPending && 'pending'}
```
Comment (PKulkoRaccoonGang) [inline src/schedule-and-details/index.jsx:311]: The props `state` of the [StatefulButton component](https://paragon-openedx.netlify.app/components/statefulbutton/#props-api) must accept a `string`, please fix it

File: src/assets/scss/_form.scss [hunk 2]
```diff
@@ -27,3 +27,55 @@
...
+  .datepicker-custom-control {
+    display: block;
+    width: 100%;
+    font-size: $input-font-size;
+    font-weight: $input-font-weight;
+    line-height: $input-line-height;
+    background: $input-bg;
+    border-color: $input-border-color;
+    border-width: $input-border-width;
+    box-shadow: $input-box-shadow;
```
Comment (PKulkoRaccoonGang) [inline src/assets/scss/_form.scss:43]: Let`s try this
```suggestion
    border: $input-border-width solid $input-border-color $input-box-shadow;
```

Comment (ruzniaievdm) [reply src/assets/scss/_form.scss]: fixed, thx!

File: src/schedule-and-details/index.jsx (hunk 1 above)
Comment (ruzniaievdm) [reply src/schedule-and-details/index.jsx:311]: let's do it next PR with fix console error

**Review by sendr — APPROVED** [review]

**Pull request merged**